### ISSUE TITLE
Icing tool

### ISSUE THREAD

**Issue by MastPeace** [DLR-SL/CPACS-IFARX#13]
Here I want to write a short description of my tool and also start a discussion about where should I write my outputs.

I've done the rough calculations of ElectroThermal Ice Protection System (ET IPS). It's modern and good choice for composite wings. As a prototype BOEING 787 IPS was taken. As you can see on the image below it consists of 2 areas: continuously heated and cyclically heated. This made for power economy.
![изображение](https://user-images.githubusercontent.com/56656378/67040334-f05c9d80-f12b-11e9-92b7-0a38583f01ef.png)
So. At this moment my tool can calculate power required to protect a wing with 2D airfoil taken at the middle of the wing. Icing conditions were taken from CS-25. And I can implement this calculations for wing, htp and vtp. The results are:
![изображение](https://user-images.githubusercontent.com/56656378/67043035-cad29280-f131-11e9-8c18-df0a917a920c.png)
For example 787 needs about 75kW 
Also the estimation of IPS weight was made, but it's very-very rough if we can use it. For example:
![изображение](https://user-images.githubusercontent.com/56656378/67043045-d45bfa80-f131-11e9-990b-c54c04792b3e.png)

So, can we use these results somehow?

Further plans: 1. Put the tool into RCE (just need some time)
2. calculate wing area more accurately. make calculations for each segment
3. Use better method to calculate collection efficiency

**Comment by godaygo** [comment]
@MastPeace Nice work! Just to understand, what you mean by:

> calculate wing area more accurately. make calculations for each segment

This refers to the surface being washed (wetted area) or something else?

**Comment by MastPeace** [comment]
@godaygo 
> @MastPeace Nice work! Just to understand, what you mean by:
> 
> > calculate wing area more accurately. make calculations for each segment
> 
> This refers to the surface being washed (wetted area) or something else?

Yes, it refers to more accurate calculation of wing wetted area. As the first step I want to do calculations for each segment of the wing. And as the second - use another method of calculating collection efficiency

**Comment by MastPeace** [comment]
Hi all!
As it was discussed on today's teleconference I tried to vary parameters in the input file to OpenAD.
So... The scheme of workflow 
![изображение](https://user-images.githubusercontent.com/56656378/67893255-4b988200-fb67-11e9-89c1-4ffcc03fc0cf.png)
I tried to vary AR. The values were 8, 10.3 and 12. The results are:
![изображение](https://user-images.githubusercontent.com/56656378/67893375-8a2e3c80-fb67-11e9-88da-10069b50ebc7.png)
![изображение](https://user-images.githubusercontent.com/56656378/67893392-92867780-fb67-11e9-8d13-353fea30d012.png)
![изображение](https://user-images.githubusercontent.com/56656378/67893411-9a461c00-fb67-11e9-9cd7-fa0a70ecfc57.png)
As you can see the power is really sensitive to the changes, and the mass is not (this is really rough estimation, as I said before)

P.S. Also I have a question. What is the material of the wing? I thought it is made of composite and choose Electro Thermal IPS. Is it true?
